File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/photos/PhotoViewUseCase.java
package nl.acidcats.tumblrlikes.core.usecases.photos;

import rx.Observable;

/**
 * Created on 17/10/2018.
 */
public interface PhotoViewUseCase {

    Observable<Boolean> startPhotoView(long id, long currentTime);

    Observable<Boolean> endPhotoView(long id, long currentTime);
}

File: datalib/src/main/kotlin/nl/acidcats/tumblrlikes/core/usecases/photos/PhotoViewUseCase.kt
package nl.acidcats.tumblrlikes.core.usecases.photos

import rx.Observable

/**
 * Created on 17/10/2018.
 */
interface PhotoViewUseCase {

    fun startPhotoView(id: Long, currentTime: Long): Observable<Boolean>

    fun endPhotoView(id: Long, currentTime: Long): Observable<Boolean>
}
